react/features/dynamic-branding/components/native/BrandingImageBackground.tsx

```diff
@@ -41,7 +41,7 @@ const BrandingImageBackground: React.FC<Props> = ({ uri }:Props) => {
                     // with the smallest Y value of the viewport.
                     preserveAspectRatio = 'xMinYMin'
                     style = { styles.brandingImageBackgroundSvg as StyleProp<ViewStyle> }
-                    uri = { imgSrc }
+                    uri = { imgSrc ?? null }
```

Comment:
Is this necessary? Does it complain in case imgSrc is undefined ?

Reply:
Yes. It says it should be either string or null